### PR TITLE
Python Widget Store

### PR DESCRIPTION
**Pull request by kraftp**

### CODE REVIEW

File: python/widget-store/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+You can also visit the [DBOS Cloud Console](https://console.dbos.dev/) to see your app's status and logs.
+
+### Running Locally
+
+To run this app locally, you need a Postgres database.
+If you have Docker, you can start one with:
+
+```shell
+export PGPASSWORD=dbos
+python3 start_postgres_docker.py
```
Comment (qianl15) [inline python/widget-store/README.md:30]: Add a start_postgres_docker.py file in this folder?

File: python/widget-store/widget_store/main.py [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+
+
+@dbos.transaction()
+def update_order_status(order_id: str, status: int) -> None:
+    DBOS.sql_session.execute(
+        orders.update().where(orders.c.order_id == order_id).values(order_status=status)
+    )
+
+
+@dbos.transaction()
```
Comment (chuck-dbos) [inline python/widget-store/widget_store/main.py:154]: Do we have readonly transactions?

**Comment by chuck-dbos** [comment]
I didn't run it, probably should, but code looks plausible...

File: python/widget-store/widget_store/main.py (hunk 2 above)
Comment (kraftp) [reply python/widget-store/widget_store/main.py:154]: Not yet

**Review by chuck-dbos — APPROVED** [review]

**Pull request merged**